### PR TITLE
JS, python and sql instructions

### PR DESCRIPTION
**Pull request by anuveyatsu**
Created `_instructions.html` template for our instructions on using Data Packages. We will need to move it to somewhere in logic part of the app so we can write Markdown, but for now, I have written it in HTML. 
This PR also includes:
- added 'codehilite' extension to Markdown processor;
- added `input` tag into allowed list of tags;
- added `checkbox` attribute into allowed list of attributes.

### CODE REVIEW

File: app/templates/_instructions.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+<span class="c1">#Get Data Package</span>
+datapackage <span class="o">&amp;</span>lt<span class="p">;</span>
+<span class="o">-</span> datapkg_read<span class="p">(</span>
+<span class="s">"https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"</span><span class="p">)</span>
+
+<span class="c1">#Package info</span>
+<span class="kp">print</span><span class="p">(</span>datapackage<span class="p">)</span>
+
+<span class="c1">#Open actual data in RStudio Viewer</span>
```
Comment (zelima) [inline app/templates/_instructions.html]: there are no space between comment and code snippets except `#Open actual data in RStudio Viewer ` and `View(datapackage$data$"demo-resource")`

image

Lets stay consistent and either add for all of them or remove (prob remove)

File: app/templates/_instructions.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+</pre>
+</div>
+{%- endmacro %}
+
+{% macro pandas(dataset) -%}
+<p>Tested with Python 3.5.2</p>
+
+<p>To generate Pandas data frames based on JSON Table Schema descriptors we have to install <code>jsontableschema-pandas</code> plugin.
+To load resources from a data package as Pandas data frames use <code>datapackage.push_datapackage</code> function.
+*Storage* works as a container for Pandas data frames.</p>
```
Comment (zelima) [inline app/templates/_instructions.html]: is this supposed to be italic? cause it actually  is not Eg:
image

File: app/templates/_instructions.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+
+<span class="nx">fetchDataPackageAndData</span><span class="p">(</span><span class="s1">'https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json'</span><span class="p">);</span>
+</pre>
+</div>
+
+<p>Our JavaScript is written using ES6 features. We are using <code>node.js v7.4.0</code> and passing <code>--harmony</code> option to enable ES6:</p>
+<p>
+<div class="highlight">
+<pre>$ node --harmony index.js</pre>
```
Comment (zelima) [inline app/templates/_instructions.html]: This is a bit unclear for me should I install or no? If yes why it is at the end of the page?

File: app/templates/_instructions.html [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+<div class="highlight">
+<pre>
+<span></span><span class="kn">import</span> <span class="nn">datapackage</span>
+<span class="kn">from</span> <span class="nn">sqlalchemy</span> <span class="kn">import</span> <span class="n">create_engine</span>
+
+<span class="n">data_url</span> <span class="o">=</span> <span class="s1">'https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json'</span>
+<span class="n">engine</span> <span class="o">=</span> <span class="n">create_engine</span><span class="p">(</span><span class="s1">'sqlite:///:memory:'</span><span class="p">)</span>
+<span class="c1"># to load Data Package into storage</span>
+<span class="n">storage</span> <span class="o">=</span> <span class="n">datapackage</span><span class="o">.</span><span class="n">push_datapackage</span><span class="p">(</span><span class="n">data_url</span><span class="p">,</span> <span class="s1">'sql'</span><span class="p">,</span> <span class="n">engine</span><span class="o">=</span><span class="n">engine</span><span class="p">)</span>
+<span class="c1"># to see datasets in this package</span>
```
Comment (zelima) [inline app/templates/_instructions.html]: Again let's stay consistent - there is no space between comment and previous code in Pandas and sql  instruction, but there is spaces in pyhon, js and R code snippets. Choose one if them. (think spaced one is better) Eg: 
This one is better
image
then this 
image

Same with sql

File: app/templates/_instructions.html [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+install_github<span class="p">(</span><span class="s">"ropenscilabs/jsonvalidate"</span><span class="p">)</span>
+install_github<span class="p">(</span><span class="s">"ropenscilabs/datapkg"</span><span class="p">)</span>
+
+<span class="c1">#Load client</span>
+<span class="kn">library</span><span class="p">(</span>datapkg<span class="p">)</span>
+
+<span class="c1">#Get Data Package</span>
+datapackage <span class="o">&amp;</span>lt<span class="p">;</span>
+<span class="o">-</span> datapkg_read<span class="p">(</span>
+<span class="s">"https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"</span><span class="p">)</span>
```
Comment (anuveyatsu) [inline app/templates/_instructions.html]: @zelima in R, it is supposed to be so. See: https://github.com/ropenscilabs/datapkg

But I will fix new line issue, that's definitely not correct.

File: app/templates/_instructions.html (hunk 1 above)
Comment (anuveyatsu) [inline app/templates/_instructions.html]: good point, thanks

File: app/templates/_instructions.html (hunk 2 above)
Comment (anuveyatsu) [inline app/templates/_instructions.html]: will fix

File: app/templates/_instructions.html (hunk 3 above)
Comment (anuveyatsu) [inline app/templates/_instructions.html]: @zelima no, this is when you run the script using node.js

**Pull request merged**